config/config.exs

```diff
@@ -28,13 +28,22 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:messages_api_user, :user_id]
 
+config :logger,
+  backends: [:console, Sentry.LoggerBackend]
+
 config :ueberauth, Ueberauth,
   providers: [
     cognito: {SignsUi.Ueberauth.Strategy.Fake, []}
   ]
 
 config :phoenix, :json_library, Jason
 
+config :sentry,
+  enable_source_code_context: true,
+  root_source_code_path: [File.cwd!()],
```

Comment:
The key should be plural: `root_source_code_paths`. (this is still the deprecated singular form)

Reply:
🤦‍♀️ my b